### PR TITLE
add meta api for serving

### PR DESCRIPTION
**Pull request by u2takey**
Add meta data api for model metadata, which is useful for checking model info.
Related rpc service is defined here: https://github.com/tensorflow/serving/blob/master/tensorflow_serving/apis/prediction_service.proto#L29

This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/800)

### CODE REVIEW

File: components/k8s-model-server/http-proxy/server.py [hunk 1]
```diff
@@ -192,6 +193,17 @@ def get_signature(signature_map, signature_name=None):
     raise KeyError("No signature found for signature key %s." % signature_name)
 
 
+class MetaHandler(tornado.web.RequestHandler):
+  """
+  Meta Hanlder proxy return Model metadata. Defined here https://github.com/tensorflow/serving/blob/master/tensorflow_serving/apis/prediction_service.proto#L29
```
Comment (pdmack) [inline components/k8s-model-server/http-proxy/server.py]: "Handler"

**Comment by pdmack** [comment]
Hmmm, tests are failing but your branch is _way_ out of date. As a sanity check, @u2takey can you rebase with upstream master please?

**Comment by jlewi** [comment]
Please update the PR description as well and explain what a MetaAPI is for?

/assign @lluunn

File: components/k8s-model-server/http-proxy/server.py [hunk 2]
```diff
@@ -192,9 +193,20 @@ def get_signature(signature_map, signature_name=None):
     raise KeyError("No signature found for signature key %s." % signature_name)
 
 
+class MetaHandler(tornado.web.RequestHandler):
+  """
+  Meta Handler proxy return Model metadata. Defined here https://github.com/tensorflow/serving/blob/master/tensorflow_serving/apis/prediction_service.proto#L29
```
Comment (lluunn) [inline components/k8s-model-server/http-proxy/server.py]: Currently it only supports signature map. We should add a comment here.

**Pull request merged**